siriuspy/siriuspy/devices/sofb.py

```diff
@@ -51,19 +51,32 @@ class DEVICES:
         'MeasRespMatKickCV-SP', 'MeasRespMatKickCV-RB',
         'MeasRespMatWait-SP', 'MeasRespMatWait-RB',
         'NrSingValues-Mon', 'MinSingValue-SP', 'MinSingValue-RB',
-        # properties used only for ring-type accelerators:
+        )
+    # properties used only for ring-type accelerators:
+    _propty_ring_tmpl = (
         'MTurnAcquire-Cmd',
         'SlowOrbX-Mon', 'SlowOrbY-Mon',
         'MTurnSum-Mon', 'MTurnOrbX-Mon', 'MTurnOrbY-Mon',
         'MTurnIdxOrbX-Mon', 'MTurnIdxOrbY-Mon', 'MTurnIdxSum-Mon',
         'MTurnTime-Mon',
-        # properties used only for sirius:
+        )
+    # properties used only for sirius:
+    _propty_si_tmpl = (
         'KickRF-Mon',
         'DeltaKickRF-Mon', 'DeltaKickRF-SP',
         'MaxDeltaKickRF-RB', 'MaxDeltaKickRF-SP',
         'ManCorrGainRF-SP', 'ManCorrGainRF-RB',
         'MeasRespMatKickRF-SP', 'MeasRespMatKickRF-RB',
         'RFEnbl-Sel', 'RFEnbl-Sts',
+        'DriveFreqDivisor-SP', 'DriveFreqDivisor-RB', 'DriveFrequency-Mon',
```

Comment:
Should some of these PVs be archived? I think our policy should be to archive everything by default. Not archiving PVs should be for particular reasons (too little useful info for too much storage used, for example).